src/main/resources/templates/smallfull/accountsReferenceDate.html

```diff
@@ -0,0 +1,80 @@
+
+<!DOCTYPE html>
+
+<html
+        xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout" xmlns:th="http://www.w3.org/1999/xhtml"
+        layout:decorate="~{layouts/baseLayout}">
+
+<head>
+    <title id="page-title-heading">
+        Confirm your new company account date
+    </title>
+</head>
+<div id="accounts-reference-date-main-content" layout:fragment="content">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <form id="accounts-reference-date-form" class="form" th:action="@{''}" th:object="${accountsReferenceDate}" method="post">
+
+                <div th:replace="fragments/globalErrors :: globalErrors"></div>
+
+                <h1 id="page-title" class="govuk-heading-l">
+                    <span>Confirm your new company account date</span>
+                </h1>
+                <span id="new-company-account-date-hint" class="govuk-hint">
+                  You can change the end date of the period you're filing for by up to 7 days.
+                </span>
+
+
+                <div class="govuk-form-group"
+                     th:classappend="${#fields.hasErrors('chosenDate')} ? 'govuk-form-group--error' : ''">
+                    <div class="govuk-radios">
+                        <div class="govuk-grid-row">
+                            <div class="govuk-grid-column-one-half">
+                                <th:block th:each="pastDate, stat : ${accountsReferenceDate.pastDates}">
+                                    <div class="govuk-radios__item">
+                                        <input class="govuk-radios__input" type="radio"
+                                               th:id="${stat.index}"
```

Comment:
This id is going to match the one for future dates (i.e. there will be 2 radios on the page with `id="0"`)